docs/cadc/cadc.rst

```diff
@@ -124,10 +127,10 @@ resolved. Instead it is matched against the target name in the CADC metadata.
   >>> from astroquery.cadc import Cadc
   >>>
   >>> cadc = Cadc()
-  >>> result = cadc.query_name('M31)
+  >>> result = cadc.query_name('M31')
```

Comment:
Nice catch :sunglasses:

Reply:
oh, we should totally figure out this remote doctesting. Maybe at scipy sprinting...